### PR TITLE
removes v1 krypto from local server

### PR DESCRIPTION
**Pull request by James-Pickett**

### CODE REVIEW

File: ee/localserver/krypto-ec-middleware.go [hunk 1]
```diff
@@ -169,3 +169,25 @@ func extractChallenge(r *http.Request) (*challenge.OuterChallenge, error) {
 
 	return challenge.UnmarshalChallenge(decoded)
 }
+
+type bufferedHttpResponse struct {
+	header http.Header
```
Comment (James-Pickett) [inline ee/localserver/krypto-ec-middleware.go:174]: could probably simplify by embedding http.ResponseWriter and override Write() to write to buf

**Review by RebeccaMahany — DISMISSED** [review]

File: ee/localserver/server.go [hunk 2]
```diff
@@ -83,37 +83,18 @@ func New(logger log.Logger, db *bbolt.DB, kolideServer string) (*localServer, er
...
-	rsaAuthedMux.HandleFunc("/ping", pongHandler)
-	rsaAuthedMux.Handle("/id", kbm.Wrap(ls.requestIdHandler()))
-	rsaAuthedMux.Handle("/id.png", kbm.WrapPng(ls.requestIdHandler()))
-
-	// Setup the v2 protocol wraps
 	ecKryptoMiddleware := newKryptoEcMiddleware(ls.logger, ls.myLocalDbSigner, ls.myLocalHardwareSigner, *ls.serverEcKey)
-
 	ecAuthedMux := http.NewServeMux()
+	ecAuthedMux.HandleFunc("/", http.NotFound)
+	ecAuthedMux.HandleFunc("/ping", pongHandler)
```
Comment (directionless) [inline ee/localserver/server.go:89]: You can drop these. We don't use them, and they didn't work with the EC middleware.

**Review by directionless — APPROVED** [review]
So much code, gone... But still, good to have it cleaner.

**Review by seejdev — APPROVED** [review]

**Review by directionless — DISMISSED** [review]
Might need the not found. Not sure

**Review by directionless — APPROVED** [review]

**Review by seejdev — APPROVED** [review]

**Pull request merged**